src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -251,7 +272,41 @@ export function transformSearchToolAddress({ address1, address2, address3, city,
     state: state && state.slice(0, searchToolSchoolAddressFields.state.maxLength),
     postalCode: zip && zip.slice(0, searchToolSchoolAddressFields.postalCode.maxLength),
   };
+}
 
+function addPrefilledFlagsToFormData(formData) {
+  const newFormData = { ...formData };
+  Object.keys(prefillFlagsToFieldsMap).forEach(flag => {
+    prefillFlagsToFieldsMap[flag].forEach(field => {
```

Comment:
`find()` would probably work better here. You could separate the predicate from the assignment and the function would only loop as many times as needed (i.e. this only needs to loop over the predicate- not the assignment).

Reply:
maybe `some()` is better. https://stackoverflow.com/a/25926600

Reply:
I new this smelled wrong because I wanted to avoid the inability to exit out of a `forEach` early, but I don't think I even knew about `find` until now. Thanks for the tip.